test/test.js

```diff
@@ -46,7 +102,125 @@ describe('/GET comment', () => {
       .end((err, res) => {
         res.should.have.status(400);
         res.body.msg.should.be.a('string');
-        // res.body.msg.should.equal('no song with id 101');
+        done();
+      });
+  });
+});
+
+describe('Test CRUD Operations for comments', () => {
+  after((done) => {
+    chai
+      .request(app)
+      .post('/comment/1')
+      .send({
+        user_id: userIdHolder,
+        song_id: songIdHolder,
+        content: contentHolder,
+        time_stamp: timeStampHolder,
+      })
+      .end((err, res) => {
+        expect(res).to.have.status(201);
+        expect(err).to.be.null;
+        res.body.success.should.equal(true);
+        done();
+      });
+  });
+
+  it('PUT should be able to edit an existing comment', (done) => {
```

Comment:
See my comment to server/index.js regarding having an `id`-specific `PUT` route.

Reply:
Makes sense